integration/bmp/task_cmd_test.go

```diff
@@ -0,0 +1,33 @@
+package bmp_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gexec"
+
+	. "github.com/cloudfoundry-community/bosh-softlayer-tools/integration/test_helpers"
+)
+
+var _ = Describe("`$bmp task --task_id` integration tests", func() {
+	var session *Session
+
+	BeforeEach(func() {
+		session = RunBmpTarget()
+		Expect(session.ExitCode()).To(Equal(0))
+
+		session = RunBmpLogin()
+		Expect(session.ExitCode()).To(Equal(0))
+	})
+
+	Describe("$bmp task --task_id", func() {
+		Context("execute bmp task with default level", func() {
+			BeforeEach(func() {
+				session = RunBmp("task", "--task_id=1")
+			})
+
+			It("returns 0", func() {
+				Expect(session.ExitCode()).To(Equal(0))
+			})
```

Comment:
Something like: `Expect(session).To(Say("..."))`, of course replace `...` with an expected string in output